### DISCUSSION TITLE
[Community Call] Agenda - June 7, 2022

### DISCUSSION THREAD

**Discussion by natemoo-re** [withastro/roadmap#206] [Announcements]
## To Discuss

- **Announcement** [v1.0 Release Post](https://astro.build/blog/astro-1-release-update/)
- **Announcement** Hackathon Update @bholmesdev 
- **Issues** No issues needing discussion
- **Governance** [PR has passing votes](https://github.com/withastro/astro/pull/3490)
- **RFC** Expose [`injectRoute`](https://github.com/withastro/rfcs/pull/204) hook

_If there is something that you would like to see discussed, please comment below. We will prioritize items already on the agenda, but if there is time to spare we will open up the call to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions)._

**Comment by delucis** [comment]
# Astro Community Call Notes
7 Jun 22

[Discord thread](https://discord.com/channels/830184174198718474/983789010314993705)

## Agenda items

- **Nate:** 1.0 Release Update — deadline pushed back to ensure improved DX https://astro.build/blog/astro-1-release-update/
- **Ben:** Hackathon update. AWARDS CEREMONY! YouTube stream Wednesday 8 June. Live announcements for The Astros with Ben & Fuzzy
- **Nate:** no issues currently need discussion! 🥳
- **Nate:** Governance changes PR ([astro#3490](https://github.com/withastro/astro/pull/3490))
  - Core no longer a purely technical role
  - More formalisation around different contributor teams (docs, i18n, etc.)
  - New Technical Steering Committee structure to steward astro’s core technical changes
- `injectRoute` hook — [RFC PR](https://github.com/withastro/astro/pull/3457)
  - Some questions to be answered in the future to support passing configuration from integrations to injected route. For now let’s try the API as-is and see what more we need as people build.
  - Consensus reached and RFC PR merged
  - [Docs PR](https://github.com/withastro/docs/pull/704) ready for review

## AOB

- **Ben:** reported on isSelfAccepting’s undead status. Potentially co-ordinating fix with Vite team (prompted by Okiki)
- **Okiki:** will dependency changes in Vite v3 cause any issues for Astro?
   **Nate:** shouldn’t do
- **@enteleform** HMR roadmap status?
   **Nate & Tony:** work on this is ongoing and v1 should feature improved HMR
- **Fuzzy:** flag idea of better error messages — also an ongoing effort
